x-pack/plugins/task_manager/server/task_pool.ts

```diff
@@ -148,7 +149,11 @@ export class TaskPool {
   private cancelExpiredTasks() {
     for (const task of this.running) {
       if (task.isExpired) {
-        this.logger.debug(`Cancelling expired task ${task.toString()}.`);
+        this.logger.warn(
```

Comment:
love it!  This would be great to have in the event log, once we figure out how to add TM events to it ...